build.gradle

```diff
@@ -10,8 +10,13 @@ plugins {
 	id 'nebula.lint' version '8.3.1'
 	id 'nebula.source-jar' version '7.0.1' apply false
 	id 'maven-publish'
+        id 'com.github.sherter.google-java-format' version '0.8'
 }
 
+repositories {
+  // to get the google-java-format jar and dependencies
+  mavenCentral()
```

Comment:
If adding `mavenCentral()` repository here, then no need to also add it later for just the Java subprojects. Look for and remove the now-redundant `mavenCentral()` that appears [later in this same file](#diff-c197962302397baf3a4cc36463dce5eaR78).